frontend/src/components/ResultPage.vue

```diff
@@ -119,131 +148,111 @@ function sendFeedback (isCorrect: boolean) {
       <div
         class="result-image"
         :style="{backgroundImage:`url(${img})`}"
-      />
+      ></div>
       <div class="fr-tile fr-enlarge-link mb-3">
-        <h2 class="fr-tile__title px-2">
-          <div v-if="confidenceLevel === 'low'">
-            <div class="fr-tile__body">
-              <DsfrTag
-                class="fr-tag--sm error-tag"
-                label="Indice de fiabilité insuffisant"
-              />
+        <h2 class="fr-tile__title px-2" />
+        <div v-if="confidenceLevel === 'low'">
+          <div class="fr-tile__body">
+            <DsfrTag
+              class="fr-tag--sm error-tag"
+              label="Indice de fiabilité insuffisant"
+            />
+          </div>
+          <p class="category fr-callout__title mt-3">
+            <img
+              class="px-2"
+              src="@/assets/guide-identification/icones/gun.jpg"
+              alt=""
```

Comment:
Update this design when confidence level is low

Reply:
Done ! 
<img width="479" alt="image" src="https://github.com/datalab-mi/Basegun/assets/17519882/c53eaa83-5c63-42ef-8a49-c1e883de8961">